snakegame/snakegame/models/Point.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Point.swift
+//  snakegame
+//
+//  Created by Vlad Z on 21/01/2024.
+//
+
+import Foundation
+
+struct Point: Codable {
+    var x: Double
+    var y: Double
+}
```

Comment:
I've worked under the impression that the position of things will be at integer values rather than doubles. What is correct?

Reply:
Good catch! Doubles make no sense here.